### DISCUSSION TITLE
UNKNOWN: unkonwn error, read

### DISCUSSION THREAD

**Discussion by zzhu9271** [jgraph/drawio#3503] [Q&A]
drawio version: 20.8.10 windows
Report unknown error while open drawio file which is created in another laptop,  it was synced through onedrive.
Is there anyone knowing how to solve it? Thanks in advance!
![image](https://user-images.githubusercontent.com/130546605/231399350-41cedbc0-fa8c-427f-bcaf-72d8652fb4dc.png)

**Comment by ZainabAl-khabori** [comment]
Same. One was a Mac and the other Windows, I initially thought it was because of the difference in operating systems, but that doesn't seem to be the case.
